GameServerLib/GameObjects/Spell/Missile/SpellCircleMissile.cs

```diff
@@ -187,60 +187,81 @@ protected override bool IsValidTarget(IAttackableUnit unit)
                 return false;
```

Comment:
Why this function not simply call the base class `SpellMissile`? What are the differences?